common/index.ts

```diff
@@ -23,15 +22,41 @@ export enum ServiceEndpoints {
   Experiments = '/api/relevancy/experiments',
 }
 
+/**
+ * BACKEND SEARCH RELEVANCE APIs
+ */
 export enum BackendEndpoints {
   QuerySets = '/_plugins/search_relevance/query_sets',
   SearchConfigurations = '/_plugins/search_relevance/search_configurations',
   Judgments = '/_plugins/search_relevance/judgments',
   Experiments = '/_plugins/search_relevance/experiments',
 }
 
+/**
+ * OPEN SEARCH CORE APIs
+ */
 export const SEARCH_API = '/_search';
 
+/**
+ * Node APIs
+ */
+export const BASE_NODE_API_PATH = '/api/relevancy';
```

Comment:
I removed all these constants in a recent commit. Is there a reason why you introduced them again? Perhaps a git related issue?